examples/games/snake/Exceptions.ark

```diff
@@ -0,0 +1,9 @@
+{
+    (import "Either.ark")
+    
+    (let throw left)
+    (let return right)
+    (let try (fun (either continue handle)
+        (either handle continue)
+    ))
+}
```

Comment:
same here, avoid copying the file of the standard library just to "make it work" ; by sending the rights flags to the virtual machine, it can find the standard library